core/src/main/java/com/sequenceiq/cloudbreak/service/OperationRetryService.java

```diff
@@ -43,14 +43,15 @@ public void retry(Stack stack) {
         }
 
         Cluster cluster = stack.getCluster();
-        if (Status.AVAILABLE.equals(stack.getStatus()) && Status.AVAILABLE.equals(cluster.getStatus())) {
-            LOGGER.info("Retry cannot be performed, cluster created successfully. stackId: {}", stack.getId());
-            throw new BadRequestException("Retry cannot be performed, cluster created successfully.");
+        if (Status.CREATE_FAILED.equals(stack.getStatus()) ||
```

Comment:
This looks me a hard coded behavior, and only works for only a few operation. Would be better to make it configurable per flow